src/SourceBuild/tarball/content/tools-local/tasks/Microsoft.DotNet.SourceBuild.Tasks.XPlat/FileUtilities.cs

```diff
@@ -0,0 +1,68 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+namespace Microsoft.DotNet.Build.Tasks
+{
+    public static class FileUtilities
+    {
+        public const string CR = "\r";
+        public const string CRLF = "\r\n";
+        public const string LF = "\n";
+
+        public static string DetectNewLineChars(string source)
+        {
+            if (source == null)
+            {
+                throw new ArgumentNullException(nameof(source));
+            }
+
+            if (source.Contains(CRLF))
+            {
+                return CRLF;
+            }
+            else if (source.Contains(LF))
+            {
+                return LF;
+            }
+            else if (source.Contains(CR))
+            {
+                return CR;
+            }
+            else if (source.Contains(LF))
```

Comment:
Is there a reason that LF is checked twice?

Reply:
I copied when I meant to cut and paste 😸.  Removed.